server/app/services/scheduler/error.rb

```diff
@@ -1,5 +1,3 @@
 module Scheduler
-  class Error < StandardError
-
-  end
+  Error = Class.new(StandardError)
```

Comment:
This is so weird... how does Ruby know that `Error.new.class.name => "Error"` :question: